openerp/tests/common.py

```diff
@@ -347,6 +348,7 @@ def phantom_run(self, cmd, timeout):
             self._wait_remaining_requests()
             # we ignore phantomjs return code as we kill it as soon as we have ok
             _logger.info("phantom_run execution finished")
+            self.assertIs(result, True, 'PhantomJS test failed.')
```

Comment:
While we're changing it, perhaps it would be nice to have a more specific error message to distinguish it from the other phantomjs failures. Maybe something like this:

```python
self.assertTrue(result, "PhantomJS test completed without reporting success, "
    "the log may contain errors or hints.")
```